apps/src/netsim/NetSimStatusPanel.js

```diff
@@ -50,9 +50,6 @@ NetSimStatusPanel.inherits(NetSimPanel);
 
 /**
  * @param {Object} [data]
- * @param {boolean} [data.isConnected] - Whether the local client is connected
- *        to a remote node
```

Comment:
`isConnected` is no longer relevant, because the status panel is _only_ shown when you are connected.